### PR TITLE
cembrapay icon

### PR DESCRIPTION
**Pull request by manishgautamwork**
## Why are you adding this icons?
I'm adding/updating this icon. because I'm creating a payment provider and this icon will be used in this app.

#### Help us identify yourself
- [x] I'm working/collaborating with the brand directly and they have provided the icons.
- [x] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [x] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I have attached a screenshot comparison with the example icon provided in guidelines
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

#### Tips how to create a screenshot
We have found free online SVG editor https://www.freecodeformat.com/svg-editor.php very useful to create one. Here is a sample code for you to verify that you icon appears properly along side the placeholder.

### Icon Screenshot
cembrapay

```
<!-- Change background color if needed to showcase your icon better -->
<style> body { background: black; } </style>

<!-- CembraPay Icon -->
<svg role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-cembrapay" xmlns="http://www.w3.org/2000/svg"><title id="pi-cembrapay">cembrapay</title><path style="stroke:none;fill-rule:nonzero;fill:#baec9f;fill-opacity:1" d="M1.563 20.73C.867 22.102 0 20.387 0 17.648V3.563C0 .824.867-.891 1.563.48l4.289 8.477-4.29 11.777zm0 0"/><path style="stroke:none;fill-rule:nonzero;fill:#004d8c;fill-opacity:1" d="M35.352 22.5c.214-.129.386-.313.511-.55.13-.243.23-.548.309-.927.07-.351.156-.859.223-1.445h-.157c-.39-3.195-.816-7.418-1.101-11.039a.985.985 0 0 1 .472-.148l.832 8.742c.04.433.086.933.106 1.41h.027c.028-.477.067-.977.114-1.434l.84-8.722c.124 0 .343.043.472.152 0 0-.945 9.531-1.113 11.035-.172 1.508-.309 2.328-.54 3.086-.218.715-.523 1.16-.917 1.34-.043-.5-.075-1.078-.075-1.5zM32.039 9.219c0-3.176-.742-4.758-1.5-4.758-.402 0-.691.105-1.148.297v14.82h.52v-5.433h.597c.738 0 1.531-1.711 1.531-4.926zm-.531 0c0 2.238-.496 3.433-1.074 3.433h-.524V6.04c.254-.086.465-.129.617-.129.477 0 .985 1.094.985 3.309zm3.555 8.91h-.344c-.14 0-.188-.192-.188-.703v-8.14c-.265-.727-.734-1.044-1.152-1.044-.316 0-.566.149-.84.426 0 .488.027 1.125.063 1.535.18-.281.492-.469.765-.469.258 0 .543.211.7.508v3.09l-.688.11c-.766.105-1.156 1.019-1.156 3.343 0 2.047.386 3.004.937 3.004.324 0 .617-.3.922-1.148h.027c.106.87.325 1.148.575 1.148.246 0 .379-.344.379-1.172zm-.997-.66c-.214.531-.535.918-.816.918-.367 0-.52-.492-.52-1.707 0-1.383.153-1.918.63-1.98l.706-.106zM5.711 8.73l-3.86-4.8C1.2 3.113.532 5.027.653 7.375l.715 13.824c.117 2.348.938 3.367 1.469 1.836l3.144-9.023c.536-1.535.383-4.465-.27-5.282zm4.266 11.067c-1.274 0-1.957-2.66-1.957-7.723 0-5.347.84-7.68 1.89-7.68.406 0 .95.286 1.195.711a75.796 75.796 0 0 1-.132 2.93c-.297-.488-.672-.758-1.012-.758-.504 0-.906 1-.906 4.774 0 3.773.308 4.86.945 4.86.371 0 .805-.466 1.059-1.044.066.89.117 1.887.136 2.93-.324.668-.836 1-1.218 1zm4.222-5.125h-1.87v.133c0 1.465.167 2.172.745 2.172.301 0 .707-.309.914-.707.078.773.153 1.886.192 2.816-.301.465-.813.71-1.117.71-1.07 0-1.676-2-1.676-5.858 0-4.086.625-6.172 1.492-6.172.95 0 1.344 2.066 1.344 5.148 0 .492-.004 1.223-.024 1.754zm-.86-2.598c0-1.375-.14-2.066-.472-2.066-.332 0-.531.8-.531 2.664h1c.004-.133.004-.488.004-.598zm5 7.5v-7.879c0-.82-.077-1.308-.398-1.308-.199 0-.378.222-.523.597v8.59h-.875V11.72c0-.867-.074-1.332-.371-1.332-.188 0-.39.222-.55.597v8.59h-.88V8.098c.254-.11.598-.176.88-.176v1.152h.042c.32-.976.547-1.308.863-1.308.32 0 .59.379.735 1.464.332-1.152.61-1.464.96-1.464.571 0 .993 1.066.993 3.593v8.215zm2.895.223c-.539 0-1.05-.445-1.41-1.422V3.262c.254-.114.598-.2.875-.2v5.926h.047c.254-.957.492-1.222.793-1.222.688 0 1.23 1.554 1.23 5.546 0 3.997-.562 6.485-1.535 6.485zm.024-9.414c-.211 0-.418.246-.559.691v5.727c.117.285.309.418.48.418.497 0 .657-1.043.657-3.637 0-2.598-.195-3.2-.578-3.2zm3.832.625a.572.572 0 0 0-.32-.133c-.239 0-.422.152-.63.645v8.054h-.874V8.098c.25-.11.593-.176.875-.176v1.484h.043c.32-1.508.527-1.64.742-1.64.058 0 .168.046.238.132 0 1.043-.031 2.086-.074 3.11zm2.707 8.789c-.293 0-.516-.266-.629-1.2h-.04c-.265.801-.554 1.2-.89 1.2-.59 0-1.02-1.024-1.02-3.352 0-2.554.462-3.422 1.274-3.53l.531-.067v-1.973c-.16-.336-.394-.445-.628-.445-.274 0-.57.199-.766.488-.05-.73-.11-1.73-.125-2.684.3-.289.625-.464.96-.464.509 0 1.06.418 1.395 1.351v6.902c0 .579.047.778.203.778h.352v1c0 1.531-.27 1.996-.617 1.996zm-.774-4.973-.53.067c-.286.046-.4.312-.4 1.355 0 .89.114 1.242.36 1.242.188 0 .43-.152.57-.531zm0 0"/></svg>

```

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)

What's the expected date of this change to deploy on Shopify?

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -2210,3 +2210,7 @@
   name: qliro
   label: Qliro
   group: other
+-
+  name: cembrapay
```
Comment (Lovedanihonjin) [inline db/payment_icons.yml:2214]: Any reason why CembraPay is not using any capitalization?

File: app/assets/images/payment_icons/cembrapay.svg
Comment (Lovedanihonjin) [inline app/assets/images/payment_icons/cembrapay.svg:1]: Please make sure the SVG file follows the following guidelines: https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#guidelines-for-new-icons

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Requested changes in the SVG and .yml file.

File: db/payment_icons.yml (hunk 1 above)
Comment (manishgautamwork) [reply db/payment_icons.yml:2214]: I have updated it.

File: app/assets/images/payment_icons/cembrapay.svg
Comment (manishgautamwork) [reply app/assets/images/payment_icons/cembrapay.svg:1]: Yes, the icon is following the guidelines.

**Comment by manishgautamwork** [comment]
@Lovedanihonjin, Changes are made as per the requirements. Please have a look. So we can proceed further.

Thanks

**Comment by manishgautamwork** [comment]
@Lovedanihonjin, Any Update on it?

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Please revise the icon. Background, borders and border radiuses are all missing from your SVG image. The icon also seems to be squished/resized incorrectly.

Attaching appearance requirements: https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#appearance
Screenshot 2024-05-09 at 9 15 22 AM

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Thanks for the commit. Please use the original vector file for the CembraPay. Also, the inner and outer border radius and border color is incorrect. 

Screenshot 2024-05-13 at 8 15 14 AM
Screenshot 2024-05-13 at 8 18 04 AM

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Thanks for your updates. Can you please use the correct icon for cembrapay? 
Screenshot 2024-05-16 at 10 59 19 AM

**Review by Lovedanihonjin — APPROVED** [review]
Looks good thank you!

**Comment by manishgautamwork** [comment]
@Lovedanihonjin,
Thanks for the approval, please let me know when this will be released.

**Pull request merged**